instance/multi.go

```diff
@@ -47,6 +50,8 @@ func (m *MultitenantInstancer) Get(instanceID flux.InstanceID) (*Instance, error
 		creds,
 		log.NewContext(instanceLogger).With("component", "registry"),
 		m.RegistryMetrics.WithInstanceID(instanceID),
+		m.MemcacheClient,
```

Comment:
Memcache client should be a middleware for the docker client. I can refactor this in a subsequent commit with my other registry changes.